### DISCUSSION TITLE
SOC 2 Compliance?

### DISCUSSION THREAD

**Discussion by jasonshave** [datalust/seq-tickets#2256] [Help]
I'd like to make a recommendation to use SEC in my organization but I'm told we can only deal with SOC 2 compliant products. Any plans in the works to achieve this certification?

**Comment by nblumhardt** [comment]
Hi Jason, thanks for dropping us a line!

Seq is self-hosted software that customers run on their own infrastructure, so we don't currently maintain a SOC 2, which is strongly oriented towards service providers ("service organizations").

Just to make an analogy, organizations don't generally require SOC 2 from operating system or database vendors where that code runs on the organization's self-hosted infrastructure. Seq is in the same category here.

Datalust does not store or process application data on its customers' behalf. You can find some additional info in https://datalust.co/privacy.

Hope this helps,
Nick

**Comment by jasonshave** [comment]
That makes perfect sense. I'll relay the info. Cheers and keep up the great work!
